#!/bin/sh /usr/share/dpatch/dpatch-run
## 02-host-module.dpatch by Torsten Werner <twerner@debian.org>
##
## DP: Added KERN_DIR to the HostDrivers Makefile.

@DPATCH@

diff -Naurp virtualbox-ose-1.5.4-dfsg.orig/src/VBox/HostDrivers/Support/linux/Makefile virtualbox-ose-1.5.4-dfsg/src/VBox/HostDrivers/Support/linux/Makefile
--- virtualbox-ose-1.5.4-dfsg.orig/src/VBox/HostDrivers/Support/linux/Makefile	2007-10-12 21:19:22.000000000 +0000
+++ virtualbox-ose-1.5.4-dfsg/src/VBox/HostDrivers/Support/linux/Makefile	2007-12-30 06:29:05.000000000 +0000
@@ -59,6 +59,7 @@ endif
 ifneq ($(MAKECMDGOALS),clean)
 
 # kernel base directory
+KERN_DIR := $(KDIR)
 ifndef KERN_DIR
  # build for the current kernel, version check
  KERN_DIR := /lib/modules/$(shell uname -r)/build
